<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTime</class>
 <widget class="QWidget" name="DateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="CustomTimeCtrl.qrc">
    <normaloff>:/agenda.png</normaloff>:/agenda.png</iconset>
  </property>
  <property name="statusTip">
   <string>Date Time Input</string>
  </property>
  <property name="whatsThis">
   <string>Widget to Input DateTime</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Input the date here</string>
            </property>
            <property name="statusTip">
             <string>Input date</string>
            </property>
            <property name="whatsThis">
             <string>Widget to input date</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="lbDate">
               <property name="text">
                <string>Date</string>
               </property>
               <property name="buddy">
                <cstring>dateEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="lbDateFormat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-&gt; DATE FORMAT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Input the time here</string>
            </property>
            <property name="statusTip">
             <string>Input time</string>
            </property>
            <property name="whatsThis">
             <string>Widget to input time</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="lbTime">
               <property name="text">
                <string>Time</string>
               </property>
               <property name="buddy">
                <cstring>timeEdit</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QTimeEdit" name="timeEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="lbTimeFormat">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>-&gt; TIME FORMAT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupHasTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkTime">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Has time</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="CustomTimeCtrl.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>DateTime</receiver>
   <slot>setHasTime(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DateTime</sender>
   <signal>isAuto(bool)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DateTime</sender>
   <signal>isAuto(bool)</signal>
   <receiver>timeEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>isAuto(bool)</signal>
  <signal>isDateTime(bool)</signal>
  <slot>setHasTime(bool)</slot>
 </slots>
</ui>
